.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     hooks:
       - id: yamllint
   - repo: https://github.com/awslabs/cfn-python-lint
-    rev: v0.52.0
+    rev: v0.61.5
```

Comment:
This needed to be upgraded because our integration test sam functions had a now-outdated python version runtime (3.6), but the latest version (3.9) wasn't accepted by our old version of cfn-lint.